### ISSUE TITLE
Cannot view txt file inside zip file

### ISSUE THREAD

**Issue by Andreas0602** [mucommander/mucommander#825]
Download the zip file from #824 and double-click it to go inside the zip file. Then select the file "iso-8859-1.txt" and press F3. Result: Error message "Unable to view file." This problem did not occur with muCommander v1.1.0.

<img width="360" alt="Screenshot_2022-12-30_15-24-26" src="https://user-images.githubusercontent.com/79788982/210080901-03ffd098-e105-479e-913e-34e7f63d9d48.png">

**Comment by pskowronek** [comment]
```
>> 2023-01-02 19:20:56.254 [Thread-8] ERROR com.mucommander.ui.viewer.FileFrame:80 - Exception caught
java.lang.NullPointerException: null
        at com.mucommander.commons.io.StreamUtils.readUpTo(StreamUtils.java:404)
        at com.mucommander.commons.io.StreamUtils.readUpTo(StreamUtils.java:381)
        at com.mucommander.commons.io.EncodingDetector.detectEncoding(EncodingDetector.java:162)
        at com.mucommander.viewer.text.TextViewer.startEditing(TextViewer.java:143)
        at com.mucommander.viewer.text.TextEditor.open(TextEditor.java:244)
        at com.mucommander.ui.viewer.FileEditorPresenter.switchFileEditor(FileEditorPresenter.java:199)
        at com.mucommander.ui.viewer.FileEditorPresenter.show(FileEditorPresenter.java:177)
        at com.mucommander.ui.viewer.FilePresenter.open(FilePresenter.java:160)
        at com.mucommander.ui.viewer.FileFrame$1.getTargetComponent(FileFrame.java:77)
        at com.mucommander.ui.layout.AsyncPanel$2.run(AsyncPanel.java:109)
```

**Comment by pskowronek** [comment]
Provided fix in  #830 PR
